packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -586,7 +592,7 @@ class DatePicker extends React.Component {
   }
 
   onInputKeyDown = (event) => {
-    if(event.key === 'Enter') {
+    if(event.keyCode === 13 || event.keyCode === 32) {
```

Comment:
What is the reason for adding the `SPACE` key here? Should this be updated to get the keyCode from `nativeEvent ` and use `KEY_RETURN` and `KEY_SPACE` from keycode-js similar to how [other places](https://github.com/cerner/terra-framework/blob/ca8fde828d309978e69ab801d59044f5e71626b0/packages/terra-application-navigation/src/common/_PopupMenuListItem.jsx#L8-L9) use it?

Reply:
Border was not getting added when opening via space, so added space key also.([comment](https://github.com/cerner/terra-framework/pull/1216#issuecomment-675417899)). Updated to keycode eb41920